### DISCUSSION TITLE
Looking for a NorthPass LMS source (and willing to pay for it)

### DISCUSSION THREAD

**Discussion by aidanlister** [airbytehq/airbyte#39357] [New Connector Request]
We're after a source for https://www.northpass.com/

We'd like to grab the information of people taking courses, e.g. name, email, course taken and the date.

**Comment by afonso-pereira** [comment]
I can tackle it, there is an endpoint on Northpass for this, however, I dont have an account with data to test it.
Do you have any way of testing it (or giving me access to a data source for development)?

Edit: I got the request answer examples from northpass, but further testing is needed.

**Comment by natikgadzhi** [comment]
It shipped and is available on both Cloud and OSS! 

I'll leave this open. Try it out and tell us if you have any issues with it.

[a comment by an automated account is not shown]
